src/main/java/seedu/duke/Finux.java

```diff
@@ -0,0 +1,85 @@
+package seedu.duke;
+
+import seedu.duke.command.CreditScoreMap;
+import seedu.duke.command.Command;
+import seedu.duke.command.CommandHandler;
+import seedu.duke.command.ExitCommand;
+import seedu.duke.exception.FileLoadingException;
+import seedu.duke.parser.ParserHandler;
+import seedu.duke.record.RecordList;
+import seedu.duke.storage.Storage;
+import seedu.duke.ui.Ui;
+
+import java.util.ArrayList;
+
+import static seedu.duke.common.Constant.FINUX_LOGGER;
+
+public class Finux {
+    private Ui ui;
+    private RecordList records;
+    private Storage storage;
+    private CommandHandler commandHandler;
+    private ParserHandler parserHandler;
+    private CreditScoreMap creditScoreMap;
+
+    /**
+     * Main entry-point for Finux application.
+     */
+    public static void main(String[] args) {
+        new Finux().run();
+    }
+
+    /**
+     * Runner for the FINUX Application.
+     */
+    private void run() {
+        start();
+        commandLooper();
+        end();
+    }
+
+    /**
+     * Starts the main application.
+     */
+    private void start() {
+        try {
+            ui = new Ui();
+            storage = new Storage();
+            parserHandler = new ParserHandler();
+            commandHandler = new CommandHandler();
+            storage.loadFile();
```

Comment:
Should there be a call to the loadFile() method of Storage here? Also, the line for creditScoreMap seems to be omitted from the diagram as well. Was it probably intended?
![Presentation1](https://user-images.githubusercontent.com/19249948/113076601-00721b00-9202-11eb-8461-37303e2eb9cf.png)

Reply:
Thanks for pointing this out! We have changed it.